index.js

```diff
@@ -0,0 +1,66 @@
+import {
+    form,
+    listbtn,
+    addlistbtn,
+    contactbtn,
+    list,
+    newbooks,
+    contact,
+    date,
+} from "./modules/book.js";
+
+import Book from "./modules/class.js";
+
+const bookList = new Book();
```

Comment:
- [ ] [REQUIRED] Good job so far! When I try to `remove` one of the `added options`, the remaining added options remain removed. Please correct this 🙏🏼 

![mobile](https://github.com/BenMKT/Awsome-Books-with-ES6/assets/63915024/cf420766-57e8-4b45-87c3-3184f3f5c7d3)